python/pydantic_core/_pydantic_core.pyi

```diff
@@ -345,6 +346,7 @@ class SchemaSerializer:
             fallback: A function to call when an unknown value is encountered,
                 if `None` a [`PydanticSerializationError`][pydantic_core.PydanticSerializationError] error is raised.
             serialize_as_any: Whether to serialize fields with duck-typing serialization behavior.
+            polymorphic_serialization: Whether to override configured model and dataclass polymorphic serialization for this call.
```

Comment:
Maybe we should follow the existing pattern for the description? (That is, other parameters don't mention this "override" behavior of the configuration. Maybe in the actual docstring of the method, mention that _these parameters override the corresponding configuration_ or similar).